packages/devkit/src/integrationmodel.js

```diff
@@ -12,7 +12,8 @@ import warnIcon from '../styles/icons/general/warn_icon.svg';  //eslint-disable-
  * @typedef {Object} IntegrationModelProperties
  * @property {string} configurationService - Configuration service path.
  * This parameter is needed to determine all services paths.
- * @property {HTMLElement} integrationModelProperties.target - HTML target.
+ * @property {HTMLElement} 
+ * .target - HTML target.
```

Comment:
```suggestion
 * @property {HTMLElement} integrationModelProperties.target - HTML target.
```

Este cambio creo que ha sido por error al refactorizar la variable, habría que deshacerlo!